examples/demo-simple-stf/src/lib.rs

```diff
@@ -26,7 +33,7 @@ pub enum ApplySlotResult {
     Success,
 }
 
-impl<Vm: Zkvm, Cond: ValidityCondition, B: BlobReaderTrait> StateTransitionFunction<Vm, B>
+impl<Vm: ZkVerifier, Cond: ValidityCondition, Da: DaSpec> StateTransitionFunction<Vm, Da>
```

Comment:
DaSpec has associated ValidityCondition, so it is possible to decrease used generics:

```suggestion
impl<Vm: ZkVerifier, Da: DaSpec> StateTransitionFunction<Vm, Da>
```